airgun/views/partitiontable.py

```diff
@@ -0,0 +1,76 @@
+from widgetastic.widget import (
+    Checkbox,
+    ConditionalSwitchableView,
+    Text,
+    TextInput,
+    View,
+)
+
+from airgun.views.common import (
+    BaseLoggedInView,
+    SatTab,
+    SearchableViewMixin,
+)
+from airgun.widgets import (
+    ACEEditor,
+    ActionsDropdown,
+    FilteredDropdown,
+    MultiSelect,
+)
+
+
+class PartitionTableView(BaseLoggedInView, SearchableViewMixin):
+
+    title = Text("//h1[text()='Partition Tables']")
+    new = Text("//a[contains(@href, '/ptables/new')]")
+    actions = ActionsDropdown("//td//div[contains(@class, 'btn-group')]")
```

Comment:
Please take a brief look at #89 and its changes to entities.
We're now finally working with entities table and widgets inside its columns, instead of picking the first button/link/actionsdropdown available on page.
Basically you'll have to update:
* PartitionTableView to have table widget with column widgets defined, instead of `actions` and `edit`
* EditPartitionTable and ClonePartitionTable step() to use column widgets
* lock(), unlock() and delete() entity methods to use column widgets

PR #89 contains a lot of examples for every single entity available, so it should be pretty straightforward, but feel free to ping me in case of any questions.